guide/src/continuous-integration.md

```diff
@@ -50,10 +50,10 @@ deploy:
   provider: pages
   skip-cleanup: true
   github-token: $GITHUB_TOKEN
-  local-dir: path/to/mybook/book
+  local-dir: book
   keep-history: false
   on:
-    branch: master
+    branch: main
```

Comment:
The text above still references the "master" branch.

Reply:
Done